package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "eslint packages --report-unused-disable-directives",
     "lint:jss": "stylelint './packages/**/src/**/*.styles.js'",
     "postpublish": "git add . && git commit -m 'Publish: automate readme and changelog update' --no-verify && git push origin master -f --no-verify",
-    "prepare": "yarn cleanup && lerna run pretranspile && yarn transpile && yarn readme",
+    "prepare": "yarn cleanup && lerna run pretranspile && yarn transpile && yarn readme && yarn types",
```

Comment:
I gonna piggyback while you are here, can you unified the name convention for the reader, changelog and the types generation? 

maybe generateReadME, generateTypeDefinitions and generateChangeLOG? 🙏